src/styles/base.js

```diff
@@ -47,6 +47,8 @@ export const titleColorLT = '#29323C';
 export const subtextColor = '#6A7485';
 export const subtextColorLT = '#A0A7B4';
 export const subtextColorActive = '#FFFFFF';
+export const subtextBorderColorActive = '#FFFFFF';
+export const subtextWidth = '30px';
```

Comment:
what is it? why is a `text` need width

Reply:
thos belongs to the width of the panel toggles